00-programming-fundamentals/exercises/github-verification.md

```diff
@@ -0,0 +1,92 @@
+# Verify Your Github Account
+
+## Overview
+
+We want to verify that we have the right github name for you and that your git and the Github website communicate properly.  There are some brief commands explained, but we will get into those commands in depth later, just follow along.
+
+## 1.  Find The Sample Repository
+
+Go to [the sample repository](https://github.com/Ada-C9/github_checkup).
+
+![Sample Repo](./images/sample-repo.png)
+
+## 2.  Then Click On The `Fork` Button
+
+![fork](./images/fork.png)
+
+This will make a copy of the repository onto your github account.  
+
+## 3.  Copy the Clone Link
+
+Next you need to get a link to connect git and github.  To get it Click on the Clone or Download Button and select
+
+![Clone or Download](./images/clone-or-download.png)
```

Comment:
Could you please update this image to show the HTTPS link, since they won't have ssh keys set up yet?